src/guide/advanced/hot-reload.md

```diff
@@ -0,0 +1,88 @@
+# How to hot reload Iroha in a Docker container
```

Comment:
Keep this in the repo.

Reply:
kept in the repo, check: https://github.com/hyperledger/iroha/pull/2879